data/Makefile.am

```diff
@@ -75,6 +74,10 @@ filetypes = \
 	filedefs/filetypes.yaml \
 	filedefs/filetypes.zephir
 
+# generated filetypes
+filetypes_nodist = \
+	filedefs/filetypes.python
```

Comment:
shouldn't it be `filetypes.python.in` thats nodist?

Reply:
It needs `filetypes.python.in` to generate `filetypes.python` when the user of the dist configures it.

Reply:
oh, ok, it needs to be generated on the target